### PR TITLE
[BULK] - DocuTune remediation - Sensitive terms with GUIDs (part 9)

### PR DESCRIPTION
**Pull request by meganbradley**
Applying sensitive terms with GUID changes as part of Content SFI and outlined in [Overview - Writing content securely - Platform Manual](https://review.learn.microsoft.com/en-us/help/platform/security-reference?branch=main). Changes are part of the Microsoft-wide SFI effort. Point of contact: @CelesteDG

DocuTune v1.5.2.0
CorrelationId: [ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db](https://github.com/Azure/azure-docs-powershell-azuread/pulls?q=is%3Apr+ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db+)

#docutune

### CODE REVIEW

**Comment by meganbradley** [comment]
#docutune-review

**Comment by msewaweru** [comment]
Trigger rebuild.

File: azureadps-2.0-preview/AzureAD/Remove-AzureADMSApplicationKey.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ syntaxes:
 examples:
 - title: 'Example 1: Removes a key credential from an application'
   code: |-
-    PS C:\>Remove-AzureADMSApplicationKey -ObjectId "3ddd22e7-a150-4bb3-b100-e410dea1cb84" -KeyId "FDA27CF-1B58-4CAE-8CE7-CD04F0AAB945" -Proof {token}
+    PS C:\>Remove-AzureADMSApplicationKey -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb" -KeyId "FDA27CF-1B58-4CAE-8CE7-CD04F0AAB945" -Proof {token}
```
Comment (msewaweru) [inline azureadps-2.0-preview/AzureAD/Remove-AzureADMSApplicationKey.yml:14]: Key ID GUID not updated.

File: azureadps-2.0-preview/AzureAD/Remove-AzureADMSFeatureRolloutPolicyDirectoryObject.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ federation).
 
 ### Example 1: Removes a group from the cloud authentication roll-out policy from Azure AD.
 ```
-PS C:\> Remove-AzureADMSFeatureRolloutPolicyDirectoryObject -Id "a03b6d9e-6654-46e6-8d0a-8ed83c675ca9" -ObjectId "0a1068c0-dbb6-4537-9db3-b48f3e31dd76"
+PS C:\> Remove-AzureADMSFeatureRolloutPolicyDirectoryObject -Id "a03b6d9e-6654-46e6-8d0a-8ed83c675ca9" -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb"
```
Comment (msewaweru) [inline azureadps-2.0-preview/AzureAD/Remove-AzureADMSFeatureRolloutPolicyDirectoryObject.md:29]: Id GUID here hasn't been updated.

File: azureadps-2.0-preview/AzureAD/Remove-AzureADMSFeatureRolloutPolicyDirectoryObject.yml [hunk 3]
```diff
@@ -9,7 +9,7 @@ syntaxes:
 examples:
 - title: 'Example 1: Removes a group from the cloud authentication roll-out policy from Azure AD.'
   code: |-
-    PS C:\> Remove-AzureADMSFeatureRolloutPolicyDirectoryObject -Id "a03b6d9e-6654-46e6-8d0a-8ed83c675ca9" -ObjectId "0a1068c0-dbb6-4537-9db3-b48f3e31dd76"
+    PS C:\> Remove-AzureADMSFeatureRolloutPolicyDirectoryObject -Id "a03b6d9e-6654-46e6-8d0a-8ed83c675ca9" -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb"
```
Comment (msewaweru) [inline azureadps-2.0-preview/AzureAD/Remove-AzureADMSFeatureRolloutPolicyDirectoryObject.yml:12]: Same as previous comment.

File: azureadps-2.0-preview/AzureAD/Restore-AzureADDeletedApplication.md [hunk 4]
```diff
@@ -33,26 +33,26 @@ ObjectId                             AppId                                Displa
 421c3f21-22b1-43ea-b438-f00bcad54bd7 f9009add-63a4-4231-9532-9bdc22742922 PowerShellGraphAPI
 4862738f-9ce9-4db6-ab55-e185049f4597 d961ff63-d659-42d5-8ca8-908b3bbb79cb WingTips
 49a8bc01-2751-450b-a2e8-b4267f609513 10d861e6-90b3-4854-a504-f656aab2a14e AzurePopulator
-79592454-dea7-4660-9d91-f1768e5055ac feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
+aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
```
Comment (msewaweru) [inline azureadps-2.0-preview/AzureAD/Restore-AzureADDeletedApplication.md:36]: Other real GUIDs have not been updated.

File: azureadps-2.0-preview/AzureAD/Restore-AzureADDeletedApplication.yml [hunk 5]
```diff
@@ -24,26 +24,26 @@ examples:
     421c3f21-22b1-43ea-b438-f00bcad54bd7 f9009add-63a4-4231-9532-9bdc22742922 PowerShellGraphAPI
     4862738f-9ce9-4db6-ab55-e185049f4597 d961ff63-d659-42d5-8ca8-908b3bbb79cb WingTips
     49a8bc01-2751-450b-a2e8-b4267f609513 10d861e6-90b3-4854-a504-f656aab2a14e AzurePopulator
-    79592454-dea7-4660-9d91-f1768e5055ac feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
+    aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
```
Comment (msewaweru) [inline azureadps-2.0-preview/AzureAD/Restore-AzureADDeletedApplication.yml:27]: Not all GUIDs have been updated.

**Review by msewaweru — CHANGES_REQUESTED** [review]
Changes requested.

File: azureadps-2.0-preview/AzureAD/Remove-AzureADMSApplicationKey.yml (hunk 1 above)
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Remove-AzureADMSApplicationKey.yml:14]:
```suggestion
    PS C:\>Remove-AzureADMSApplicationKey -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb" -KeyId "aaaaaaaa-0b0b-1c1c-2d2d-333333333333" -Proof {token}
```

File: azureadps-2.0-preview/AzureAD/Remove-AzureADMSConditionalAccessPolicy.md [hunk 6]
```diff
@@ -24,7 +24,7 @@ Conditional access policies are custom rules that define an access scenario.
 
 ### Example 1: Deletes a conditional access policy in Azure AD by PolicyId.
 ```
-PS C:\> Remove-AzureADMSConditionalAccessPolicy -PolicyId 6b5e999b-0ba8-4186-a106-e0296c1c4358
+PS C:\> Remove-AzureADMSConditionalAccessPolicy -PolicyId 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Remove-AzureADMSConditionalAccessPolicy.md:27]:
```suggestion
PS C:\> Remove-AzureADMSConditionalAccessPolicy -PolicyId 6b5e999b-0ba8-4186-a106-e0296c1c4358
```

File: azureadps-2.0-preview/AzureAD/Remove-AzureADMSConditionalAccessPolicy.yml [hunk 7]
```diff
@@ -8,7 +8,7 @@ syntaxes:
 examples:
 - title: 'Example 1: Deletes a conditional access policy in Azure AD by PolicyId.'
   code: |-
-    PS C:\> Remove-AzureADMSConditionalAccessPolicy -PolicyId 6b5e999b-0ba8-4186-a106-e0296c1c4358
+    PS C:\> Remove-AzureADMSConditionalAccessPolicy -PolicyId 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Remove-AzureADMSConditionalAccessPolicy.yml:11]:
```suggestion
    PS C:\> Remove-AzureADMSConditionalAccessPolicy -PolicyId 6b5e999b-0ba8-4186-a106-e0296c1c4358
```

File: azureadps-2.0-preview/AzureAD/Remove-AzureADMSNamedLocationPolicy.md [hunk 8]
```diff
@@ -24,7 +24,7 @@ Named locations are custom rules that define network locations which can then be
 
 ### Example 1: Deletes a named location policy in Azure AD with given PolicyId.
 ```
-PS C:\> Remove-AzureADMSNamedLocationPolicy -PolicyId 76fdfd4d-bd80-4c1e-8fd4-6abf49d121fe
+PS C:\> Remove-AzureADMSNamedLocationPolicy -PolicyId 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Remove-AzureADMSNamedLocationPolicy.md:27]:
```suggestion
PS C:\> Remove-AzureADMSNamedLocationPolicy -PolicyId 76fdfd4d-bd80-4c1e-8fd4-6abf49d121fe
```

File: azureadps-2.0-preview/AzureAD/Remove-AzureADMSNamedLocationPolicy.yml [hunk 9]
```diff
@@ -8,7 +8,7 @@ syntaxes:
 examples:
 - title: 'Example 1: Deletes a named location policy in Azure AD with given PolicyId.'
   code: |-
-    PS C:\> Remove-AzureADMSNamedLocationPolicy -PolicyId 76fdfd4d-bd80-4c1e-8fd4-6abf49d121fe
+    PS C:\> Remove-AzureADMSNamedLocationPolicy -PolicyId 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Remove-AzureADMSNamedLocationPolicy.yml:11]:
```suggestion
    PS C:\> Remove-AzureADMSNamedLocationPolicy -PolicyId 76fdfd4d-bd80-4c1e-8fd4-6abf49d121fe
```

File: azureadps-2.0-preview/AzureAD/Remove-AzureADMSPermissionGrantConditionSet.md [hunk 10]
```diff
@@ -24,7 +24,7 @@ Delete an Azure Active Directory permission grant condition set object by id.
 
 ### Example 1: Delete a permission grant condition set from a policy
 ```
-PS C:\>Remove-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "excludes" -Id "665a9903-0398-48ab-b4e9-7a570d468b66"
+PS C:\>Remove-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "excludes" -Id "1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5"
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Remove-AzureADMSPermissionGrantConditionSet.md:27]:
```suggestion
PS C:\>Remove-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "excludes" -Id "665a9903-0398-48ab-b4e9-7a570d468b66"
```

File: azureadps-2.0-preview/AzureAD/Remove-AzureADMSPermissionGrantConditionSet.yml [hunk 11]
```diff
@@ -18,7 +18,7 @@ syntaxes:
 examples:
 - title: 'Example 1: Delete a permission grant condition set from a policy'
   code: |-
-    PS C:\>Remove-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "excludes" -Id "665a9903-0398-48ab-b4e9-7a570d468b66"
+    PS C:\>Remove-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "excludes" -Id "1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5"
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Remove-AzureADMSPermissionGrantConditionSet.yml:21]:
```suggestion
    PS C:\>Remove-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "excludes" -Id "665a9903-0398-48ab-b4e9-7a570d468b66"
```

File: azureadps-2.0-preview/AzureAD/Restore-AzureADDeletedApplication.md (hunk 4 above)
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Restore-AzureADDeletedApplication.md:36]:
```suggestion
79592454-dea7-4660-9d91-f1768e5055ac feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
```

File: azureadps-2.0-preview/AzureAD/Restore-AzureADDeletedApplication.yml (hunk 5 above)
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Restore-AzureADDeletedApplication.yml:27]:
```suggestion
    79592454-dea7-4660-9d91-f1768e5055ac feabcdd1-711a-4d55-ad5e-0d0577aaaa5e analog
```

**Review by cilwerner — CHANGES_REQUESTED** [review]
@msewaweru a few changes here, please commit the suggestions and merge the PR

**Review by msewaweru — APPROVED** [review]

**Pull request merged**